### DISCUSSION TITLE
Does fare checker work with companion passes?

### DISCUSSION THREAD

**Discussion by himnim** [jdholtz/auto-southwest-check-in#87] [Q&A] [answered]
The "[Fare Check](https://github.com/jdholtz/auto-southwest-check-in/blob/master/CONFIGURATION.md#fare-check)" section states that companion passes are not supported, but [this](https://github.com/jdholtz/auto-southwest-check-in/pull/73#issuecomment-1518686834) comment states that companion passes were tested - I assumed this to mean that it was working but wanted to confirm as I'm guessing that the testing was not successful based on no further updates after [this](https://github.com/jdholtz/auto-southwest-check-in/pull/73#issuecomment-1512009645) comment.

**Comment by jdholtz** [comment, marked as answer]
Yes, companion passes are not supported. What I mean by them being tested is that the script correctly detects when a reservation is a companion pass and skips it. 

The reason for them not being supported is that Southwest doesn’t let you check fares for a companion flight without first canceling the companion, checking, and then re-adding the companion. I don’t think this would be a good idea to do at all, so therefore it was decided to not support them. If you have any other ideas though, feel free to let me know.

**Reply by himnim** [reply]
Would there be a way to manually provide that info? For example, once I book the flight and add a companion, I can manually input the points/cash cost of the flight at the time of booking and then monitor for differences based on that?

Edit: I see you already addressed this [here](https://github.com/jdholtz/auto-southwest-check-in/pull/78), so you can disregard - thanks!

**Reply by fusionneo** [reply]
> Would there be a way to manually provide that info? For example, once I book the flight and add a companion, I can manually input the points/cash cost of the flight at the time of booking and then monitor for differences based on that?
> 
> Edit: I see you already addressed this [here](https://github.com/jdholtz/auto-southwest-check-in/pull/78), so you can disregard - thanks!

If you have the points/cash to spare, what I've done for my flights is booked a second ticket with points which I'll cancel the day before my flight to book my companion. This way I can be sure there's a ticket reserved for my companion (by me) and I'm still able to use the fare_checker to monitor flight prices.
